datafusion/proto/src/physical_plan/to_proto.rs

```diff
@@ -1014,8 +768,12 @@ impl TryFrom<&FormatOptions> for copy_to_node::FormatOptions {
             FormatOptions::PARQUET(options) => {
                 copy_to_node::FormatOptions::Parquet(options.try_into()?)
             }
-            FormatOptions::AVRO => copy_to_node::FormatOptions::Avro(AvroOptions {}),
-            FormatOptions::ARROW => copy_to_node::FormatOptions::Arrow(ArrowOptions {}),
+            FormatOptions::AVRO => {
+                copy_to_node::FormatOptions::Avro(protobuf_common::AvroOptions {})
+            }
+            FormatOptions::ARROW => {
+                copy_to_node::FormatOptions::Arrow(protobuf_common::ArrowOptions {})
+            }
```

Comment:
Can we re-export `protobuf_common` from `protobuf` so that import usages don't need to change?